css/src/components/popover.scss

```diff
@@ -4,6 +4,86 @@ $popover-border-radius: $border-radius !default;
 $popover-shadow: $box-shadow-heavy !default;
 $popover-width: 224px !default;
 
+$popover-caret-size: 8px;
+$popover-caret-border-size: 9px;
+$popover-caret-offset: 10px;
+
+@mixin caret($direction, $size, $color) {
```

Comment:
Have you seen this file - `mixins/chevron.scss`? Can it be re-used instead of this `caret`? If not, please move it to mixins folder

Reply:
updated